compiler-plugin-tests/spotbugs-exclude.xml

```diff
@@ -0,0 +1,27 @@
+<!--
+  ~ Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
If this is a new file we need to change the licence header's date

Reply:
I had changed that already? Can you check now?